frontend/src/app/features/work-packages/routing/partitioned-query-space-page/partitioned-query-space-page.component.html

```diff
@@ -3,7 +3,7 @@
   <div class="toolbar-container -editable">
     <div class="toolbar">
       <op-back-button *ngIf="backButtonCallback"
-                      linkClass="back-button"
+                      linkClass="back-button op-back-button_mobile-full-width"
```

Comment:
Not sure if we want to make the full-width the default, also `back-button` should probably be `op-back-button` here. Not sure how the styles work, but using a modifier without using the block doesn't make sense imo

Reply:
`op-back-button` is always applied on this component per default, the `linkClasss` elements are additional classes. Although it seems as if `back-button` is not used anywhere else. 🤷‍♀️ 
Regarding the default: I'd argue for it in the sense of consistency (and because it is technically much easier ^^). I'd change it for now and wait for a case to come up where it is perceived as weird.